src/componets/index.js

```diff
@@ -1,3 +1,5 @@
-// Exportação dos componentes
-// exemplo
-// export { default as Header } from './Header/Header';
+import IntroElasCodam from './IntroElasCodam/index.jsx';
+
+export {
+  IntroElasCodam
+};
```

Comment:
Também funciona se for assim: export { default as IntroElasCodam } from './IntroElasCodam/index.jsx';